<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSelectDialog</class>
 <widget class="QDialog" name="DataSelectDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome to RIVET!</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="computeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
  <widget class="QFrame" name="parameterFrame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>135</y>
     <width>586</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="xLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="xbinSpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>124</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ybinSpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>157</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="homDimLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Homology Degree:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="homDimSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>67</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="parameterLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="xbinLabel_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bins</string>
    </property>
   </widget>
   <widget class="QLabel" name="axesLabel">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>100</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QLabel" name="revLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xRevCheckBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>127</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="yRevCheckBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>154</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="xAxisLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>125</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yAxisLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maxDistLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dataTypeComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>33</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>points</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>points_fn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>metric</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>metric_fn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bifiltration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>firep</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>36</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtration:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterComboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>33</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>degree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>function</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="maxDistBox">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>68</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="maxDistHelp">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="functionComboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>33</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>user</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>balldensity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>knndensity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>eccentricity</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="parameterSpinBox">
    <property name="geometry">
     <rect>
      <x>516</x>
      <y>33</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>First, select a file.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fileTypeLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>You can start from a point cloud, finite metric space, bifiltration, FIRep, or a module invariants file.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data FIle</string>
    </property>
   </widget>
  </widget>
  <zorder>parameterFrame</zorder>
  <zorder>computeButton</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
